src/Uno.UI/UI/Xaml/Shapes/Polygon.Android.cs

```diff
@@ -21,6 +21,7 @@ protected override Android.Graphics.Path GetPath(Size availableSize)
 				{
 					c.LineTo(new Point(coords[i].X, coords[i].Y), true, false);
 				}
+				c.LineTo(new Point(coords[0].X, coords[0].Y), true, false);
```

Comment:
Doesn't seems related to stroke thickness.

Reply:
Yeah I noticed that Polygon shapes don't close themselves on Android, should I make a new issue and move this to a new PR?

Reply:
It's probably best, it's easier to exclude later on :)